### ISSUE TITLE
It would be nice to be able to define custom rectangular regions in AeroVal config

### ISSUE THREAD

**Issue by jgliss** [metno/pyaerocom#427]
AeroVal provides 3 options for regional aggregation:

1. default: old AeroCom rectangular regions
2. htap: HTAP binary region masks
3. countries: country based regions

However, sometimes, one might want to focus on custom regions (e.g. in the DOMOS project, the focus will be on north africa and the atlantic region to study dust transport and deposition). Thus, it would be useful to setup the configuration file in a way to specify the regions to be used and also define new regions that do not exist in the defaults.

**Comment by jgriesfeller** [comment]
In earlier years we were just defining a new region with a unique name for these cases. We could just prepend the project 'name' to the region name (e.g. DOMOS-EUR)

**Comment by jgliss** [comment]
Yes, that would work, but it would also be nice if this could be done flexibly on the user level in the config file for an experiment, so the user can define custom regions for their purposes (which would need a unique name as input plus lat0/lat1, lon0/lon1). Then the developers would not need to add custom regions to the official pyaerocom regions, in case someone wants their own region(s).

It should not be too much work to implement that kind of behaviour, and the AeroVal backend (and frontend) foresees this kind of flexibility.

**Comment by jgriesfeller** [comment]
I did not realise that you want to do that on the user level in the aeroval config file.
I agree that it would be nice to have that, but I think that in reality users of the web processing are usually also programmers of pyaerocom. Or do we expect external users to use the web processing? In any case that is nothing so important that you should spend some time on it before you leave (in my opinion).

**Comment by jgriesfeller** [comment]
For my work a custom region based on station names would be more important. Example for this is a list of long term stations. Must be variable and time frame specific though (as learned from C3S_Aerosol)

**Comment by lewisblake** [comment]
Regions in general need to be rethought. Will reference this in an upcoming GitHub Discussion...
